lambda_functions/ml_service/GetKeyPhrases.py

```diff
@@ -17,7 +17,7 @@
 def post_TopicalPageRank(language, data):
     stage = os.environ['STAGE']    
     if language == "de":
-        url="https://8wfgkfs2wc.execute-api.eu-central-1.amazonaws.com/"+stage+"/models/TopicalPageRank"
+        url="https://api."+stage+".detektivkollektiv.org/models/TopicalPageRank"
```

Comment:
This will unfortunately not work for the "prod" stage, because the stage subdomain will not be present. 
You need to catch that case with an if. See [here](https://github.com/DetektivKollektiv/python-lambda-functions/blob/6c2effb5ee8a47c5ee6c5c2a6964da148753fadd/lambda_functions/submission_service/confirm_submission.py#L25) for reference

Reply:
Plus, you are missing the `ml_model_service` path that we configured yesterday. The full path should be `api.<stage>.detektivkollektiv.org/ml_model_service/models/TopicalPageRank`.